src/_assets/scss/inc/header.scss

```diff
@@ -26,7 +26,9 @@
 		pointer-events: 0;
```

Comment:
Is `pointer-events: 0` a valid value? Should this be `none`?

Reply:
It sure isn't valid 👻 ...fixed it, thanks for spotting it.